src/main/webapp/WEB-INF/jsp/access_confirmation.jsp

```diff
@@ -11,7 +11,7 @@
   <title>OpenClinica</title>
   <link type="text/css" rel="stylesheet" href="<c:url value="/style.css"/>"/>
   <script type="text/JavaScript" language="JavaScript" src="../includes/jmesa/jquery.min.js"></script>
```

Comment:
since you have deleted the jquery.min.js file, shouldn't all the references to this file be updated to use the new version?

Reply:
jquery.min.js is not deleted, just updated (upgraded)

Reply:
from the PR, it looks like you have deleted jquery.min.js and created a new file called jquery-3.4.1.min.js.

Reply:
actually, it looks like you have two files now: jquery-3.4.1.min.js and jquery.min.js. What's the difference between these two files?

Reply:
OK I've removed the -3.4.1.min.js file and changed all references to it to the jquery.min.js